pkgs/tools/admin/swiftclient/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonApplication, fetchPypi, requests, six, pbr }:
+
+buildPythonApplication rec {
+  pname = "python-swiftclient";
+  version = "3.6.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0sv6z72zdwzwdjng0djk3l2maryn9pz3khf69yq5ig2ycz8hh0qv";
+  };
+
+  propagatedBuildInputs = [ requests six pbr ];
+
+  doCheck = false;
```

Comment:
Why have tests been disabled? 
We prefer to almost always enable tests if we can.

Reply:
Left them out because the package contains some (test dependencies)[https://github.com/openstack/python-swiftclient/blob/master/test-requirements.txt] that are not (yet) in Nixpkgs.

Reply:
I see. Can you add a comment with which ones are not in nixpkgs? 
That way someone could package them in the future if they wanted to enable tests for this.